scripts/done-task.js

```diff
@@ -14,11 +15,13 @@ function onTaskComplete(
 
   function doneTask() {
     const task = tasks[id];
-    task.doneButton = true;
-    inputElement.classList.add("done");
-    doneButton.classList.add(HIDE_CLASS);
-    editButton.classList.add(HIDE_CLASS);
+    task.done = true;
+    toggleButton(doneButton);
+    if (!editButton.classList.contains(HIDE_CLASS)) {
```

Comment:
function `toggleButton` already does the check on the `classList`, why is it checked before calling the function?